okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -136,6 +140,20 @@ public Level getLevel() {
     return level;
   }
 
+  /** Change the limitation of response and request body logs size. */
+  public HttpLoggingInterceptor setBodyLogMax(long bodyLogMax) {
+    if (bodyLogMax < 0) {
```

Comment:
I think we should be stricter here, throw for <= -2, it probably represents a coding error.